alembic/versions/fe7656c21eaa_add_remainder_of_database_models.py

```diff
@@ -0,0 +1,67 @@
+"""add remainder of database models
+
+Revision ID: fe7656c21eaa
+Revises: 5344fc3a3d3f
+Create Date: 2018-08-01 15:44:29.298131
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'fe7656c21eaa'
+down_revision = '5344fc3a3d3f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        'sources',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('uuid', sa.String(length=36), nullable=False),
+        sa.Column('journalist_designation', sa.String(length=255),
+                  nullable=False),
```

Comment:
See just below for a discussion about tag-like data modeling... again without knowing exactly what journalist designation means here, maybe something like tagging is applicable here too?

Reply:
Ahh so this is (admittedly not the clearest...) nomenclature from the SecureDrop server side: the journalist designation is the name for the source that the journalist sees, i.e. [`glued abnormality` in this screenshot](https://docs.securedrop.org/en/latest/_images/journalist-index_javascript.png)